### ISSUE TITLE
Custom borders feature request (libretro)

### ISSUE THREAD

**Issue by GBPlayer85** [LIJI32/SameBoy#552]
I would like to customize the default border. MiSTer does this by adding a ".sgb" file with the custom border. It may be done the same way adding the file to the RetroArch system folder.

Thanks!

**Comment by GBPlayer88** [comment]
This new tool by marcrobledo can generate .sgb  files from .png images.

[https://www.marcrobledo.com/super-game-boy-border-converter/](url)

**Comment by Akan9021** [comment]
Agreed. Could work as an alternative to emulators SGBC mode depending on how it interfaces with the different border modes.
